integration_tests/dbt_utils/dbt_project.yml

```
@@ -31,9 +31,6 @@ models:
       test_datediff: *not-synapse
       test_hash: *not-synapse
       test_last_day: *not-synapse
-    datetime:
-      test_date_spine: &disabled # BROKEN DUE TO MODEL DEF
-        +enabled: false
     materializations:
       test_insert_by_period: *disabled
```

my bad, can you make this change? it has to do w/ YAML anchors

```suggestion
      test_insert_by_period: &disabled
        +enabled: false
```